src/Graph/Tree.elm

```diff
@@ -1,24 +1,22 @@
 module Graph.Tree
     exposing
         ( Forest
-          -- BUILDING
```

Comment:
Removing these comments, as elm-format sorts the items in the exposing list, so it's impossible to align "section comments" with the items..